### ISSUE TITLE
Implement a Credit Screen

### ISSUE THREAD

**Issue by marwin89** [performous/performous#941]
### Suggestion

We need a credit screen to mention all the people and driving forces that worked for performous.

- Just a auto scrolling screen.
- you can leave screen again by pressing esc or enter
- Last entry in options menu

### Use case

This is so important to credit creators. 
With this screen it's more likely that developers join if there's at least a credit screen in game.

**Comment by Baklap4** [comment]
Instead of making it a screen only visible in the menu we can show it everytime on exit maybe and then auto-close after X seconds?

**Comment by marwin89** [comment]
> Instead of making it a screen only visible in the menu we can show it everytime on exit maybe and then auto-close after X seconds?

Good idea, 👋 but I'm afraid this could be to prominent and annoying over time (especially when you have to re-start often). Don't you think?
It's quite typical in games to have it in option menu or as a small ℹ️ - button on title screen. We should use best practice. Maybe we have to think about restructuring options in future. But this is out of scope of this issue.

**Comment by Baklap4** [comment]
> but I'm afraid this could be to prominent and annoying over time

Yeah that might be true however i don't usually restart the game during the day hmmm

Best practice is indeed throughout a menu under "credits" or "about us"

**Comment by twollgam** [comment]
And credits on the intro screen?

**Comment by marwin89** [comment]
> And credits on the intro screen?

This is a good idea. Maybe like this? Not sure how to place it...
![performous+ mainscreen-credits](https://github.com/performous/performous/assets/63517874/74406c33-b7a0-464b-84a7-15d16db35586)
